unime/src/lib/credentials/CredentialList.svelte

```diff
@@ -46,9 +46,12 @@
   <!-- Only show "No credentials" when there's also no favorites -->
```

Comment:
In the `if` you evaluate the **filtered** credentials. Here you evaluate **all** credentials. When there are no credentials at all, the the `/me` route does not render `CredentialList`. This part does not seem to ever be renderd.

Reply:
Will be tackled in https://github.com/impierce/identity-wallet/issues/253.